TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/CommandReference.rst

```diff
@@ -28,10 +28,10 @@ Package *TYPO3.FLOW*
 --------------------
 
 
-.. _`Flow Command Reference: TYPO3.FLOW typo3.flow:cache:flush`:
+.. _`Flow Command Reference: TYPO3.FLOW flow:cache:flush`:
 
-``typo3.flow:cache:flush``
-**************************
+``flow:cache:flush``
```

Comment:
ah, nevermind, the short form will work already.. Still, the references will probably be overridden by the automatic generation (triggered by https://github.com/neos/doctools ?)